internal/cmd/asyncapi/command_import.go

```diff
@@ -330,6 +341,41 @@ func (c *command) updateTopic(details *accountDetails, topicName string, kafkaBi
 	return nil
 }
 
+// TopicConfigs and XConfigs are both specifying Kafka configs, XConfigs is for those not specified in async api.
+// This function combines TopicConfigs and XConfigs
+func combineTopicConfigs(kafkaBinding kafkaBinding) map[string]string {
+	var configs map[string]string
+	if kafkaBinding.XConfigs != nil {
+		configs = kafkaBinding.XConfigs
+	} else {
+		configs = make(map[string]string)
+	}
+
+	// Loop through all fields of struct, no matter type
+	t := reflect.TypeOf(kafkaBinding.TopicConfigs)
+	v := reflect.ValueOf(kafkaBinding.TopicConfigs)
+	for i := 0; i < t.NumField(); i++ {
+		fieldType := t.Field(i)
+		fieldValue := v.Field(i)
+
+		var value string
+		if !fieldValue.IsNil() {
+			elem := fieldValue.Elem()
+			if elem.Kind() == reflect.Slice {
+				ss := make([]string, elem.Len())
+				for i := 0; i < elem.Len(); i++ {
+					ss[i] = elem.Index(i).String()
+				}
+				value = strings.Join(ss, ",")
+			} else {
+				value = strconv.FormatInt(elem.Int(), 10)
```

Comment:
So it looks like we're converting slices to a comma-delimited string, and we're converting integers to strings? Do we really need to use reflection here?